src/ostorlab/cli/vulnz/list/list.py

```diff
@@ -13,10 +14,37 @@
 
 @vulnz.command(name="list")
 @click.option("--scan-id", "-s", "scan_id", help="Id of the scan.", required=True)
+@click.option(
+    "--risk-rating", "-r", "risk_rating", help="Filter vulnerabilities by risk rating."
+)
+@click.option(
```

Comment:
You can create a custom type instead of two options.

https://click.palletsprojects.com/en/7.x/parameters/

Reply:
I dont think i need it here, since i have just two options. I agree with you that it might be helpful when there is a lot of options.